aptos-move/framework/aptos-stdlib/sources/cryptography/groups.move

```diff
@@ -0,0 +1,749 @@
+module aptos_std::groups {
```

Comment:
We will add some `///` top level comments to this file explaining prime-order groups & pairings a little bit.

I will do it later on.